extensions/blocks/simple-payments/edit.js

```diff
@@ -79,6 +84,20 @@ class SimplePaymentsEdit extends Component {
 			// Validate on block deselect
 			this.validateAttributes();
 		}
+
+		// currently the Gutenberg paste handler isn't bypassed for inputs with type="email" which breaks
+		// the block on cut or paste, so this prevents the cut and paste events bubbling up for now
+		// TODO: remove this and the related emailInput ref once email inputs are ignored by Gutenberg paste handler
+		if ( this.emailInput.current ) {
```

Comment:
AFAIK this is no way to check of an eventListener is already in place, so just removing first to be safe.